bridge/github/iterator.go

```diff
@@ -4,290 +4,228 @@ import (
 	"context"
 	"time"
 
+	"github.com/pkg/errors"
 	"github.com/shurcooL/githubv4"
 )
 
-type indexer struct{ index int }
-
-type issueEditIterator struct {
-	index     int
-	query     issueEditQuery
-	variables map[string]interface{}
+type iterator struct {
```

Comment:
Do you mind copying the comments from the previous version if that's still valid?

Reply:
Sure, I will add them soon!